discovery/db_discovery.go

```diff
@@ -278,7 +278,7 @@ func (dbo *DBOrchestratorPoolCache) cacheDBOrchs() error {
 		}
 		// Do not connect if URI host is not set
 		if uri.Host == "" {
-			skipped <- 1
+			errc <- fmt.Errorf("skipping orch=%v, URI not set", dbOrch.EthereumAddr)
```

Comment:
@victorges Should we throw this error, or would it be cleaner to simply display the fetched orchestrators? See @ad-astra-video's first solution (i.e. https://github.com/livepeer/go-livepeer/pull/3149/commits/6667483959e6b4ed2520a36b903f3728d0753ac7).